src/custom/components/SearchModal/ManageTokens/ManageTokensMod.tsx

```diff
@@ -44,6 +46,10 @@ export interface ManageTokensProps {
   ImportTokensRow: ({ theme, searchToken, setModalView, setImportToken }: ImportTokensRowProps) => JSX.Element
 }
 
+function checkIfSymbol(value: number | string) {
+  return typeof value === 'string' && value.length > 2 && value.length < 6 && /^[A-Z|0-9]*$/.test(value)
```

Comment:
Keep in mind there are tokens whose symbols don't use only a-z0-9

Simple example from Uniswap

![Screenshot from 2021-11-25 09-39-39](https://user-images.githubusercontent.com/43217/143483921-4e70c29b-f7cf-4a6e-bc47-47fb1132da6c.png)

The list is huge, so I've only found tokens with `$` in their symbols, but there could be more

Maybe it would be less error prone of you identify when it's not an address or partial address search?